R/plotcountrymap.R

```diff
@@ -45,14 +44,14 @@ plotcountrymap<-function(x,hatching=FALSE,...) {
     DF <- data.frame(country = countries,namedim = values,hatching=as.vector(x[,year,2]))
     dimnames(DF)[[2]][[2]] <- paste(namedim[1],substr(year,2,5))
     dimnames(DF)[[2]][[3]] <- paste(namedim[2],substr(year,2,5))
-    mapobject <- joinCountryData2Map(DF, joinCode = "ISO3",nameJoinColumn = "country")
-    mapCountryData(mapobject, nameColumnToPlot = dimnames(DF)[[2]][[2]],nameColumnToHatch=dimnames(DF)[[2]][[3]],...)
+    mapobject <- rworldmap::joinCountryData2Map(DF, joinCode = "ISO3",nameJoinColumn = "country")
```

Comment:
Please add a check whether rworldmap is available to the start of the function as described [here](https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-in-suggests-r-code)